update_database_scripts/downloadStudiesToFile.R

```diff
@@ -66,6 +66,12 @@ if (is_ebi_reachable()) {
     studies_time <- Sys.time()
     studies <- get_studies(interactive = FALSE)
     studiesTibble <- studies@studies
+    # remove newline characters
+    studiesTibble <- mutate(studiesTibble, initial_sample_size = str_replace_all(initial_sample_size, "\n", ""))
+    studiesTibble <- mutate(studiesTibble, initial_sample_size = str_replace_all(initial_sample_size, "\t", ""))
+    studiesTibble <- mutate(studiesTibble, replication_sample_size = str_replace_all(replication_sample_size, "\n", ""))
+    studiesTibble <- mutate(studiesTibble, replication_sample_size = str_replace_all(replication_sample_size, "\t", " "))
```

Comment:
Did you want this one to be replaced with a space instead of with nothing?

Reply:
Fixed